.ci/pipeline/ci.yml

```diff
@@ -104,11 +108,29 @@ jobs:
   - script: |
       .ci/scripts/describe_system.sh
     displayName: 'System info'
+  - task: Cache@2
+    inputs:
+      key: '"gcc" | "aarch64" | "openblas" | "$(OPENBLAS_VERSION)"'
```

Comment:
Yeah, that is a good point. Whilst I was here, it seemed a good opportunity to update the global definition of the TBB and OpenBLAS variables, as you suggested in your other comment.

Reply:
I've left the GCC key out, as it takes the GCC from the package manager, rather than defining a particular version, so the OS fully defines the GCC version